Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Create_a_security_pre-rule_for_EDL.yml

```diff
@@ -186,10 +186,15 @@ tasks:
       - - operator: isNotEmpty
           left:
             value:
-              simple: EDLObjectName
+              simple: inputs.EDLObjectName
             iscontext: true
           right:
             value: {}
+      - - operator: isNotEmpty
```

Comment:
Comment for task 7 - Was there a returned object?
If return yes, meaning the object returned we should create the security rule not creating the EDL object
![image](https://github.com/demisto/content/assets/61933087/4be887b8-e60b-46ab-a055-e4310b1681c5)

Reply:
Good Catch -> switching between `Yes` and `else` paths 
![image](https://github.com/demisto/content/assets/88268646/760ec50c-c480-4689-9a20-b4ce64bf6a9b)